src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandler.java

```diff
@@ -68,47 +79,46 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         SscsCaseData sscsCaseData = caseDetails.getCaseData();
 
         PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(sscsCaseData);
+
+        if (!SscsUtil.isSAndLCase(sscsCaseData)) {
+            log.info("Action postponement request: Cannot process non Scheduling & Listing Case for Case ID {}",
```

Comment:
Should say why

Reply:
The why is that the case is non-S&L, i.e gaps, right?